components/TextImage.vue

```diff
@@ -1,11 +1,13 @@
 <template>
   <div>
-    <article>
-      <div>
-        <h2>{{ blok.title }}</h2>
-        <rich-text-renderer :document="blok.description" />
+    <article class="container xl:flex l:flex">
+      <div class="w-9/12" >
+        <h2 class="w-full pb-5">{{blok.title}}</h2>
+        <div class="rich-text-renderer">
+        <rich-text-renderer class="mt-5 lg:w-1/2" :document="blok.description" />
+        </div>
       </div>
-      <figure v-if="blok.image.filename">
+      <figure class="xl:w-1/2 ml-10"  v-if="blok.image.filename"> 
```

Comment:
de margin-left wordt ook op mobiele versie toegepast waardoor de lay-out niet meer goed is. Kijk naar design en ook de spacing de tekst en image op mobiel moet nog toegepast worden

Reply:
dit is nog niet gedaan